### ISSUE TITLE
Spring Boot Redis Version

### ISSUE THREAD

**Issue by emeraldjava** [davetownsend/generator-spring#10]
@davetownsend - I created a new project which will use redis and v1.5.3.RELEASE of spring boot. It seems the redis jar has been renamed.

The build fails with this error

```
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=128m; support was removed in 8.0
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.springframework.boot:spring-boot-starter-redis:jar is missing. @ line 31, column 21
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project io.github.emeraldjava.realtimerates:realtimerates:0.0.1-SNAPSHOT (/home/pauloconnell/projects/github/realtimerates/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.springframework.boot:spring-boot-starter-redis:jar is missing. @ line 31, column 21
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

From the documentation here : https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-redis

```
Starter for using Redis key-value data store with Spring Data Redis and the Jedis client. 
Deprecated as of 1.4 in favor of spring-boot-starter-data-redis
```
 but should be updated to https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-data-redis . 

I'm creating an issue so i can submit a pull request for you.

**Comment by davetownsend** [comment]
@emeraldjava - great. there are quite a few updates I need to make as well. hopefully can find some time this weekend.

**Comment by davetownsend** [comment]
closed in PR #11
